### ISSUE TITLE
numpy-1.5.0-win32-superpack-python2.7  Installer fails on 64bit windows (Trac #1621)

### ISSUE THREAD

**Issue by numpy-gitbot** [numpy/numpy#2217]
_Original ticket http://projects.scipy.org/numpy/ticket/1621 on 2010-09-26 by trac user alxx, assigned to @rgommers._

When installing numpy-1.5.0-win32-superpack-python2.7  on windows7 64 bit, the installer can not find python2.7 in the registry.

Its a know problem with the distutils installer.
see http://bugs.python.org/issue6792
and
http://stackoverflow.com/questions/3652625/installing-setuptools-on-64-bit-windows

Need to have a separate 64 bit windows installer or look for the additional registry entries to find the install location or have a fall through that allows a user to still install even if it doesn't find python 2.7.

PyQt v4.7.7 uses the fall through option.
wxpython has a separate installer

Someone provides unofficial numpy (and scipy) binaries here   http://www.lfd.uci.edu/~gohlke/pythonlibs/
but I'd much prefer using an official package (if possible)

**Comment by adi-dhal** [comment]
Same was for me python installation was 64 bit and the path which it searched during installation was under 32 bit folder in windows registry. So either one can go creating a registry(copying the content from the existing file) file in the required path or go with python 32 bit installation.

**Comment by eric-wieser** [comment]
@charris: After #8695, this is now irrelevant, right?

**Comment by charris** [comment]
@eric-wieser Yep, we now rely on wheels. The superpacks were buggy and we no longer use them. Windows users should install with pip, or better yet, use a windows distribution such as Enthought or Anaconda.

@adi-dhal This issue should no longer be relevant for you and I am going to close it. If you have current problems please open a new issue.
